assets/models/menu_backgrounds/Comet.tscn

```diff
@@ -1,94 +1,98 @@
-[gd_scene load_steps=12 format=3 uid="uid://dokew8lhrri5"]
+[gd_scene load_steps=15 format=3 uid="uid://dokew8lhrri5"]
 
 [ext_resource type="PackedScene" uid="uid://c5ttb5il1h1ix" path="res://assets/models/IronRock1.tscn" id="1_dp368"]
-[ext_resource type="Texture2D" uid="uid://c4lakneio0v00" path="res://assets/textures/IronChunk_normal.png" id="2_7t38l"]
-[ext_resource type="Material" uid="uid://dcd6x2v3mfqfn" path="res://assets/models/menu_backgrounds/CometFireStart.tres" id="6_h1j6i"]
-[ext_resource type="Material" uid="uid://d1e2we22pcrbu" path="res://assets/models/menu_backgrounds/CometFireEnd.tres" id="7_71lin"]
-
-[sub_resource type="StandardMaterial3D" id="StandardMaterial3D_y5stp"]
-roughness = 0.72
-normal_enabled = true
-normal_texture = ExtResource("2_7t38l")
-
-[sub_resource type="QuadMesh" id="7"]
-size = Vector2(0.133, 0.271)
-center_offset = Vector3(0, 0.048, 0)
-
-[sub_resource type="Curve" id="8"]
-bake_resolution = 128
-_data = [Vector2(0, 0), 0.0, 0.0, 0, 0, Vector2(0.0896226, 1), 0.0, 0.0, 0, 0, Vector2(1, 0.00568175), 0.0, 0.0, 0, 0]
-point_count = 3
+[ext_resource type="Material" uid="uid://d1e2we22pcrbu" path="res://assets/models/menu_backgrounds/CometTailEnd.tres" id="3_3f177"]
+[ext_resource type="Material" uid="uid://dcd6x2v3mfqfn" path="res://assets/models/menu_backgrounds/CometTailStart.tres" id="6_h1j6i"]
+
+[sub_resource type="StandardMaterial3D" id="StandardMaterial3D_u4paa"]
+
+[sub_resource type="Curve" id="Curve_fhr2i"]
+_data = [Vector2(0, 0.473418), 0.0, 5.40276, 0, 0, Vector2(0.100946, 0.975316), 0.796959, 0.796959, 0, 0, Vector2(0.22082, 1), 0.0, 0.0, 0, 0, Vector2(0.413249, 0.35), -1.30411, -1.30411, 0, 0, Vector2(0.987382, 0), -0.0474222, 0.0, 0, 0]
+point_count = 5
+
+[sub_resource type="CurveTexture" id="CurveTexture_362cl"]
+curve = SubResource("Curve_fhr2i")
+
+[sub_resource type="ParticleProcessMaterial" id="ParticleProcessMaterial_8tr0e"]
+emission_shape = 2
+emission_sphere_radius = 0.03
+direction = Vector3(0, 0, 1)
+spread = 5.0
+initial_velocity_min = 0.3
+initial_velocity_max = 0.3
+gravity = Vector3(0, 0, 0)
+linear_accel_min = 2.0
+linear_accel_max = 2.0
+radial_accel_min = -0.150002
+radial_accel_max = -0.150002
+damping_min = 1.0
+damping_max = 1.0
+alpha_curve = SubResource("CurveTexture_362cl")
+
+[sub_resource type="QuadMesh" id="QuadMesh_80136"]
+material = ExtResource("6_h1j6i")
+size = Vector2(0.5, 0.45)
 
-[sub_resource type="Gradient" id="12"]
-offsets = PackedFloat32Array(0, 0.438776, 0.99095)
-colors = PackedColorArray(1, 1, 1, 0.294118, 1, 1, 1, 0, 1, 1, 1, 0)
+[sub_resource type="Curve" id="Curve_122gh"]
+resource_local_to_scene = true
```

Comment:
Do these curves have to be local to the scene? Unless these are modified during runtime this should be unnecessary and is a waste of memory and processing time. (and I didn't see any code attached to this node that could do that).

Reply:
No, that's unneeded. I just realised there's a whole load of those I missed when doing cleanup. I'll use that checkbox responsibly in future.